apps/docs/components/Navigation/Footer.tsx

```diff
@@ -11,7 +11,7 @@ const Footer = () => {
       <hr className="border-scale-400  mt-8"></hr>
       <div className="flex gap-4 items-center mt-6 justify-between">
         <div className="flex flex-col lg:flex-row gap-3 ">
-          <span className="text-xs text-scale-900">Supabase 2022</span>
+          <span className="text-xs text-scale-900">Supabase {new Date().getFullYear()}</span>
```

Comment:
not actually sure this is meant to be here at all.
usually this is a copyright reference.

Reply:
I have updated the PR to remove the year and link to the marketing page as suggested.